packages/core/src/components/overlay/overlay.tsx

```diff
@@ -264,15 +261,25 @@ export class Overlay extends React.Component<IOverlayProps, IOverlayState> {
     }
 
     private maybeRenderBackdrop() {
-        const { backdropClassName, backdropProps, hasBackdrop, isOpen } = this.props;
+        const {
+            backdropClassName,
+            backdropProps,
+            hasBackdrop,
+            isOpen,
+            transitionDuration,
+            transitionName,
+        } = this.props;
+
         if (hasBackdrop && isOpen) {
             return (
-                <div
-                    {...backdropProps}
-                    className={classNames(Classes.OVERLAY_BACKDROP, backdropClassName, backdropProps.className)}
-                    onMouseDown={this.handleBackdropMouseDown}
-                    tabIndex={this.props.canOutsideClickClose ? 0 : null}
-                />
+                <CSSTransition classNames={transitionName} timeout={transitionDuration}>
```

Comment:
_`classNames`_ 😱 
should we do this??

Reply:
that's their API. are you asking if we should do this for our own props? I don't think there's anything wrong with the current names, so would prefer to keep them